### DISCUSSION TITLE
Error: connect ETIMEDOUT 185.199.108.133:443

### DISCUSSION THREAD

**Discussion by quito96** [marlon360/rki-covid-api#376] [General]
Hallo ,

ich habe gestern noch über https://api.corona-zahlen.org/states/NW Daten geladen. Heute klappt das nicht mehr.
Auch diese Variante über den Endpoint : https://api.corona-zahlen.org/states schlägt fehl.

{"error":{"message":"An error occurred while fetching external data.","url":"https://raw.githubusercontent.com/robert-koch-institut/COVID-19-Hospitalisierungen_in_Deutschland/master/.zenodo.json","details":"connect ETIMEDOUT 185.199.108.133:443","stack":"Error: connect ETIMEDOUT 185.199.108.133:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1161:16)\n    at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17)“}}

Ist das ein Thema der api oder auf RKI Seite?

Im übrigen https://api.corona-zahlen.org/states/NW“ (mit einem Anführungszeichen) liefert Daten.

Vielen Dank und Gruß
Guido

**Comment by marlon360** [comment]
Danke fürs Melden. Jetzt sollte wieder alles laufen.

**Reply by quito96** [reply]
Ja Danke siht gut aus :-)
